mocks/prediction/predictPomodoros-event.json

```diff
@@ -0,0 +1,8 @@
+{
+  "body": "{\"taskid\":\"0db8ab10_eb99_11e8_9f63_37812029d5ca\",\"keyword\":\"bug\",\"workingdays\":\"2\"}"
```

Comment:
`"taskId":data.taskId, "taskName":data.taskName` is the data, please update the mock

Reply:
Yes .. will take care